supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesBackendLifecycleActorFactory.scala

```diff
@@ -101,7 +101,7 @@ object JesBackendLifecycleActorFactory {
       RuntimeAttributeDefinition(FailOnStderrKey, Option(staticDefaults(FailOnStderrKey)), usedInCallCaching = true),
       RuntimeAttributeDefinition(MemoryKey, Option(staticDefaults(MemoryKey)), usedInCallCaching = false),
       RuntimeAttributeDefinition(DisksKey, Option(staticDefaults(DisksKey)), usedInCallCaching = false),
-      RuntimeAttributeDefinition(ZonesKey, Option(staticDefaults(ZonesKey)), usedInCallCaching = false),
+      RuntimeAttributeDefinition(ZonesKey, None, usedInCallCaching = false),
```

Comment:
Wasn't sure if this was the right thing to do. W/ this PR there is a hardwired default to us-central1-b if no config option is supplied at all but if they do supply a config option that won't be accurate and these point to the static defaults